<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущая позиция:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="on_button_left">
       <property name="text">
        <string>Влево</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="on_button_stop">
       <property name="text">
        <string>СТОП</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="on_button_right">
       <property name="text">
        <string>Вправо</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="on_go_home">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Домой</string>
    </property>
   </widget>
   <widget class="QPushButton" name="on_run_relative_displacement">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Относительное смещение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="on_run_target_point">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Абсолютно смещение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="applyButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Применить к.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Отменить к.</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
